docs/development/extensions-core/stats.md

```diff
@@ -120,13 +132,19 @@ To acquire standard deviation from variance, user can use "stddev" post aggregat
     }
   ],
   "intervals": [
-    "2016-03-01T00:00:00.000/2013-03-20T00:00:00.000"
+    "2016-03-01/2013-03-20"
   ]
 }
 ```
 
 ### TopN query
 
+#### Druid SQL
+
+There is no equivalent SQL for this query.
```

Comment:
Looking at this, it's possible to have a topN query -- the groupBy query below with an `ORDER BY` on one dimension and a `LIMIT` clause should plan the query as a `topN`. You could verify this by running `EXPLAIN PLAN FOR`